content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than `2.155`.
+
+Since Jenkins `2.155`, it is possible to run Jenkins with Java 11.
```

Comment:
I think I would make the explanation shorter: strongly discourage users from using/staying on [2.155,2.163( when using Java 11. (Also, use LTS anyway if you're not already updating Jenkins at least twice a month, have a staging env, etc. :))

Reply:
I didn't speak about any LTS because at this time, there is none. I'd prefer we update the documentation when we have a LTS.

Reply:
Yeah, sorry for the noise, forget about my sentence between parentheses.

Reply:
I'm adding a `NOTE` to speak about LTS but without any version yet.